nuget/roundhouse/roundhouse.nuspec

```diff
@@ -12,6 +12,15 @@
     <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <tags>roundhouse db migration database migrator chucknorris</tags>
-	<iconUrl>https://raw.github.com/chucknorris/roundhouse/master/nuget/RoundhousE_Logo.NuGet.jpg</iconUrl>
+    <dependencies />
+	  <iconUrl>https://raw.github.com/chucknorris/roundhouse/master/nuget/RoundhousE_Logo.NuGet.jpg</iconUrl>
   </metadata>
+  <files>
+    <file src="$mergedExe$" target="bin\" />
+    <file src="$appConfigFile$" target="bin\" />
+    <file src="..\..\nuget\roundhouse\content\ReadMe.RoundhousE.txt" target="content\" />
+    <file src="..\..\docs\GettingStarted.docx" target="docs\" />
+    <file src="..\..\docs\legal\*" target="docs\" />

```

Comment:
Might be good to move this to next to the binaries, and move the binaries from the bin file?

Reply:
Which "this one" do you mean? The legal\* ones?

Reply:
Meaning instead of the docs folder as the location in the nupkg, the license and notice and other legal ones should be beside the binaries.

Reply:
That's an idea, but according to #274, they want the Licence one as LICENCE.txt in the tools folder (together with a VERIFICATION.txt). I see that the NOTICE file is just boiler-plate, so we don't need that, at least.

Reply:
LICENSE can be brought in as is. It doesn't actually need the .txt, but it's helpful so that it can be read as text on Chocolatey.org package page under the files section.